exporter/loggingexporter/config.go

```diff
@@ -19,7 +19,9 @@ import (
 )
 
 // Config defines configuration for logging exporter.
-// TODO: allow configuration of logging level
 type Config struct {
 	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+
+	// LogLevel defines log level of the logging exporter
```

Comment:
Can you document here as well what are the acceptable values for the loglevel?

Reply:
Updated. Thanks.